dask/bag/core.py

```diff
@@ -183,6 +183,29 @@ def finalize_item(results):
     return results[0]
 
 
+def unpack_kwargs(kwargs):
+    """ Extracts dask values from kwargs
+
+    Currently only dask.bag.Item and python literal values are supported.
+
+    Returns a merged dask graph and a list of [key, val] pairs suitable for
+    eventually constructing a dict.
+    """
+    dsk = {}
+    kw_pairs = []
+    for key, val in iteritems(kwargs):
+        if isinstance(val, Item):
+            dsk = merge(dsk, val.dask)
```

Comment:
Nitpick, could actually be `dsk.update(val.dask)` - `merge` creates a new dict and copies the old values over. Slightly more efficient, but not super important.